indexer/pkg/logutil/logutil.go

```diff
@@ -0,0 +1,33 @@
+package logutil
+
+import (
+	"os"
+	"time"
+
+	"github.com/lmittmann/tint"
+	"log/slog"
+)
+
+func SetLogLevel(level string) {
+	var logLevel slog.Level
+	switch level {
```

Comment:
This can be simplified, see: https://github.com/primev/mev-commit/blob/main/x/util/util.go#L35-L38